concent_api/verifier/tests/test_utils.py

```diff
@@ -199,3 +214,94 @@ def test_that_method_should_accept_ssim_list_and_delay_verification_result_with_
                 self.subtask_id,
                 VerificationResult.MATCH.name,
             )
+
+    def test_that_method_returns_ssim_equal_one(self):
+        ssim = compare_images(self.image_ones, self.image_ones, 'subtask_id')
+        self.assertEqual(ssim, 1.0)
+
+    @override_settings(
+        VERIFIER_MIN_SSIM=0.95
+    )
+    def test_that_method_returns_ssim_below_min_verifier_ssim_level(self):
+        ssim = compare_images(self.image_ones, self.image_zeros, 'subtask_id')
+        self.assertTrue(ssim < settings.VERIFIER_MIN_SSIM)
+
+    def test_that_method_raise_verification_error_when_images_have_diffrent_sizes(self):
+        with self.assertRaises(VerificationError):
+            compare_images(self.image_ones, self.image_diffrent_size, 'subtask_id')
+
+
+class TestGenerateFilePathMethods():
+
+    @pytest.mark.parametrize(('storage_path', 'file_name', 'expected'), [
+        ('tmp/', 'test_file.png', 'tmp/test_file.png'),
+        ('tmp/', '\\test_file.png', 'tmp/test_file.png'),
+        ('tmp/', '/test_file.png', 'tmp/test_file.png'),
+        ('tmp', '/test_file.png', 'tmp/test_file.png'),
+    ])  # pylint: disable=no-self-use
+    def test_that_method_returns_correct_verifier_storage_file_path(self, storage_path, file_name, expected):
+        with override_settings(VERIFIER_STORAGE_PATH=storage_path):
+            verifier_storage_file_path = generate_verifier_storage_file_path(file_name=file_name)
+
+            assert_that(verifier_storage_file_path).is_equal_to(expected)
+
+    @pytest.mark.parametrize(('subtask_id', 'extension', 'frame_number', 'expected'), [
+        ('subtask_id', 'PNG', '22', 'blender/verifier-output/subtask_id/subtask_id_0022.png'),
```

Comment:
actually, frame number shouldn't be string, should it?

Reply:
It shouldn't and I can delete it but this test proves that even frame which some how will be string it will still be valid to this method :)

Reply:
please remove it - tests are also a low level specs for source code. Having string passed to this function could be missleading about which argument this function normally expects...

Reply:
please remove :)